.gitignore

```diff
@@ -1,8 +1,18 @@
 # See https://help.github.com/ignore-files/ for more about ignoring files.
 
+
+# idea
+.idea/
+.idea/workspace.xml
+.idea/tasks.xml
+.idea/dictionaries
+.idea/vcs.xml
+.idea/jsLibraryMappings.xml
```

Comment:
I see some `.idea/*` items being present on this PR – probabl yadded before you updated ignore file – mind removing them?